### PR TITLE
Use executable location as working directory

### PR DESCRIPTION
**Pull request by peskdale**
Ensures config.json is found when F1viewer is called from elsewhere

### CODE REVIEW

**Comment by SoMuchForSubtlety** [comment]
why does `determineWorkingDirectory()` return `string`?

I would also suggest changing it's name to something like `moveToWorkingDirectory()` and calling it in `main()`

File: main.go [hunk 1]
```diff
@@ -55,10 +56,27 @@ var infoTable *tview.Table
 var debugText *tview.TextView
 var tree *tview.TreeView
 
+var (
+	_ = determineWorkingDirectory()
+	configFile = "config.json"
+)
+
+func determineWorkingDirectory() string {
```
Comment (SoMuchForSubtlety) [inline main.go]: why does `determineWorkingDirectory()` return string?

File: main.go [hunk 2]
```diff
@@ -55,10 +56,27 @@ var infoTable *tview.Table
 var debugText *tview.TextView
 var tree *tview.TreeView
 
+var (
+	_ = determineWorkingDirectory()
```
Comment (SoMuchForSubtlety) [inline main.go]: maybe move this to `main()`

File: main.go (hunk 1 above)
Comment (SoMuchForSubtlety) [reply main.go]: I would also suggest changing the function name to something like `moveToWorkingDirectory()`

**Comment by peskdale** [comment]
Some of my earlier code choices were based on trying to keep fast-forward working on my fork. Hopefully this is a bit tidier.

**Comment by SoMuchForSubtlety** [comment]
thank you :)

**Pull request merged**